model.py

```diff
@@ -598,6 +598,7 @@ class DataSource(Base):
     ADOBE = "Adobe DRM"
     PLYMPTON = "Plympton"
     OA_CONTENT_SERVER = "Library Simplified Open Access Content Server"
+    GOOGLE = "Google Oauth"
```

Comment:
I was modeling it after the Adobe Vendor ID, but I've also probably overthought it since there's unlikely to be multiple admin authentication processes for a library (even if they're different).